### ISSUE TITLE
Crosspost to nostr

### ISSUE THREAD

**Issue by huumn** [stackernews/stacker.news#352]
The goal is to tear down SN's garden wall like it was built by the German Democratic Republic and the war is over.

Ideal: We want stackers with a nip07 extension installed to be able to crosspost *all data* they own on SN to nostr with a user setting.

This is more of an epic than an issue. We should iterate to the ideal:

- [x] account setting for crossposting to nostr
- [x] posts are appropriately mapped to the correct nostr events
    - [x] discussion posts
    - [x] polls
    - [x] links
    - [x] bounties
- [ ] comments are appropriately mapped to the correct nostr events
- [x] account can specify which relays to use instead of relying on blastr
- [ ] send zap events on nostr too
- [ ] whatever else makes sense

There are a few unknowns here, like what to do with a comment whose parent isn't on nostr or if we should delegate posts to nostr for stacker not on nostr, but I'd guess we can figure that out as we iterate.

**Comment by TheWildHustle** [comment]
Stacker news slowly becoming a nostr client......if the user wants, sounds like a great idea

**Comment by huumn** [comment]
Yeah, I we'll get boiled like a frog into a nostr client over time.

**Comment by TheWildHustle** [comment]
Awesome job

**Comment by huumn** [comment]
closing in favor of making individual issues for enhancements to existing crossposting
